package.json

```diff
@@ -52,6 +52,7 @@
     "test": "test"
   },
   "dependencies": {
+    "@netlify/serverless-functions-api": "^1.5.1",
```

Comment:
I wonder if we should lock the version, such that the entire API surface in one version of `@netlify/functions` is immutable? Probably not a huge deal since we're also the authors of the dependency and we can control the releases we make, but wondering if it's a good caution to put in place.

Reply:
done in https://github.com/netlify/functions/pull/408/commits/423960cd91b4cbc880a61e2a7c21a338c3dd4d81